common/interfaces.go

```diff
@@ -21,4 +23,18 @@ type DbManagerInterface interface {
 	SetDbVersion(string)
 	GetDbVersion() string
 	GetKmsAttributes(tenantId string, entities ...string) map[string][]Attribute
+	GetOrgs() (orgs []string, err error)
+}
+
+type CipherManagerInterface interface {
+	AddOrgs(orgs []string)
+	// If input is encrypted, it decodes the input with base64,
+	// and then decrypt it. Otherwise, original input is returned.
+	// An encrypted input should be ciphertext prepended with algorithm. An unencrypted input can have any other format.
+	// An example input is "{AES/ECB/PKCS5Padding}2jX3V3dQ5xB9C9Zl9sqyo8pmkvVP10rkEVPVhmnLHw4=".
+	TryDecryptBase64(input string, org string) (output string, err error)
```

Comment:
why does this method have a Try prefix but the method below doesn't?

Reply:
I don't think the Try prefix is necessary as you're already returning an error variable

Reply:
`Try` means the input can be either unencrypted or encrypted. We support both encrypted/unencrypted orgs, so this `Try` meansjust pass what you have and I'll figure it out. It will try decrypting:

- return original input if unencrypted.

- return decrypted value if encrypted.